### ISSUE TITLE
Outline on extension connect buttons

### ISSUE THREAD

**Issue by kyleplo** [LLK/scratch-gui#2988]
### Expected Behavior

No outline.

### Actual Behavior

There is an outline.
![image](https://user-images.githubusercontent.com/31634240/44618716-46e08b80-a849-11e8-804b-a3cc85868641.png)

### Steps to Reproduce

Add a hardware extension and click a button.

### Operating System and Browser

Windows Chrome

**Comment by Kenny2github** [comment]
That looks like the default `:focus` styling you get on Chrome. Probably just need to add something that states
```css
:focus {
    outline: none;
}
```

**Comment by chexbox** [comment]
@Kenny2github That _is_ the default :focus styling you get on Chrome. @kyleplo must've hit the tab key or something similar to give it focus - I don't get it in Chromium

**Comment by kyleplo** [comment]
@chexbox No, it actually appears for a second just when clicked.

**Comment by dgnball** [comment]
Yeah, so this happens on Chromium on Mac when you tab about and when you click the button (which seems like expected behaviour to me). Thought I would check this out on FF which doesn't exhibit this behaviour. However, I would argue that FF is more broken because you can't tab onto the buttons like you can in Chrome which seems more useful. I guess the issue here is that the behaviour is different between FF and Chrome but the questions is, which one is correct?
